app/models/patron.rb

```diff
@@ -229,6 +229,17 @@ def can_schedule_green_access?
       (profile_key == 'CNAC' && affiliations.include?('affiliate:fellow'))
   end
 
+  def can_schedule_eal_access?
+    return unless Settings.schedule_once.eal_visits.enabled
+
+    faculty = %w[CNF MXF]
```

Comment:
This _could_ effectively be replaced with `can_schedule_green_access?` since the logic is the same (CodeClimate even complains about this). I wasn't sure if we'll end up getting shifting requirements for access, but I would be happy to refactor this a bit.  My intention is also to not have a dependency on Green being enabled in order for EAL to be enabled which would be the case if we simply relied on the `can_schedule_green_access?` accessor directly.

Reply:
I'm good keeping it separate until we know more/better.